marketing/templates/email_report.txt

```diff
@@ -0,0 +1,10 @@
+Hello from OCIM,
+
+{% if sent_emails %}I've sent today's share of emails to P&T and trail users.
```

Comment:
Since we are [checking](https://github.com/open-craft/opencraft/pull/789/files#diff-780cc6cdf2129833893b838950d6686891522ce2a81b4aa51c612e5be5c81c1aR121) if `sent_emails` exists - this check might not required